module/modmanager/manager.go

```diff
@@ -1074,6 +1075,49 @@ func (m *module) checkReady(ctx context.Context, parentAddr string, logger loggi
 	}
 }
 
+// FirstRun is runs a module-specific setup script.
+// TODO: return exit code?
+func (mgr *Manager) FirstRun(ctx context.Context, conf config.Module) error {
+	// We evaluate the Module's ExePath absolutely in the viam-server process so that
+	// setting the CWD does not cause issues with relative process names
+	firstRunPath, err := conf.EvaluateFirstRunPath(packages.LocalPackagesDir(mgr.packagesDir))
```

Comment:
we should be running FirstRun only once per module version, do we have that logic anywhere?